lib/wallet/wallet.js

```diff
@@ -2328,13 +2337,18 @@ class Wallet extends EventEmitter {
    * @returns {MTX}
    */
 
-  async makeUpdate(name, resource) {
+  async makeUpdate(name, resource, acct) {
```

Comment:
Why are we supporting individual accounts for updates, but not for the other covenants involving an owned name? I would think it would be all or nothing w.r.t. to these covenant types. Similarly, should we support individual accounts for redeems?

Reply:
Good point. I'll investigate adding account arguments to redeems. This shouldn't be an issue for register, transfer, finalize, etc. since it is impossible for more than one account to own a name -- unless we want to accept an account argument just for the sake of warning the user they think the wrong account owns a name.